game-core/build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
```

Comment:
This `buildscript` configuration (and the one in the root _build.gradle_) can probably be removed since these scripts no longer have any external dependencies.